l10n/en/navigation_refresh.ftl

```diff
@@ -7,12 +7,18 @@ navigation-refresh-mozilla = { -brand-name-mozilla }
 ## About us
 
 navigation-refresh-about-us = About us
+navigation-refresh-about-mozilla = About { -brand-name-mozilla }
 navigation-refresh-close-about-us-menu = Close About us menu
-navigation-refresh-mozilla-manifesto = { -brand-name-mozilla } manifesto
+navigation-refresh-mozilla-manifesto = The { -brand-name-mozilla } Manifesto
```

Comment:
**Issue (blocking):** these strings have already started being translated, so if they need updating we should use a new string instead of change the existing source string (else translators might miss that it has changed).

Suggest adding a `-v2` prefix to each of these string changes, so that we can fallback to the old string if it is not found for any given language:

```
{{ ftl('navigation-refresh-mozilla-manifesto-v2', fallback='navigation-refresh-mozilla-manifesto') }}
```